### ISSUE TITLE
ETH+TON dual issues 

### ISSUE THREAD

**Issue by drond1** [Lolliedieb/lolMiner-releases#1584]
Hello, sometimes TON doesn't start mining. Like on the scrshot:

![1 5 lol](https://user-images.githubusercontent.com/99704990/168446504-28fc6999-e686-4d76-a616-68b7ece64c7d.PNG)

And what is wrong with ETH hashrate on 3070TI, same settings on gminer 2.65:

![2 62](https://user-images.githubusercontent.com/99704990/168446507-786e8235-4e16-4e59-adcd-1fa0f91fc26c.PNG) 

WIN10, driver 512.15

**Comment by jgonzis** [comment]
> Hello, sometimes TON doesn't start mining. Like on the scrshot:
> 
> ![1 5 lol](https://user-images.githubusercontent.com/99704990/168446504-28fc6999-e686-4d76-a616-68b7ece64c7d.PNG)
> 
> And what is wrong with ETH hashrate on 3070TI, same settings on gminer 2.65:
> 
> ![2 62](https://user-images.githubusercontent.com/99704990/168446507-786e8235-4e16-4e59-adcd-1fa0f91fc26c.PNG)
> 
> WIN10, driver 512.15

With the 3070ti is been reported a problem ONLY in Windows and that confirms it, is not giving the proper result, same in HiveOS/MMPOS/RaveOS is working perfectly

**Comment by drond1** [comment]
> > Hello, sometimes TON doesn't start mining. Like on the scrshot:
> > ![1 5 lol](https://user-images.githubusercontent.com/99704990/168446504-28fc6999-e686-4d76-a616-68b7ece64c7d.PNG)
> > And what is wrong with ETH hashrate on 3070TI, same settings on gminer 2.65:
> > ![2 62](https://user-images.githubusercontent.com/99704990/168446507-786e8235-4e16-4e59-adcd-1fa0f91fc26c.PNG)
> > WIN10, driver 512.15
> 
> With the 3070ti is been reported a problem ONLY in Windows and that confirms it, is not giving the proper result, same in HiveOS/MMPOS/RaveOS is working perfectly

Ok, thx, will fixed on next release ?

**Comment by jgonzis** [comment]
> > > Hello, sometimes TON doesn't start mining. Like on the scrshot:
> > > ![1 5 lol](https://user-images.githubusercontent.com/99704990/168446504-28fc6999-e686-4d76-a616-68b7ece64c7d.PNG)
> > > And what is wrong with ETH hashrate on 3070TI, same settings on gminer 2.65:
> > > ![2 62](https://user-images.githubusercontent.com/99704990/168446507-786e8235-4e16-4e59-adcd-1fa0f91fc26c.PNG)
> > > WIN10, driver 512.15
> > 
> > 
> > With the 3070ti is been reported a problem ONLY in Windows and that confirms it, is not giving the proper result, same in HiveOS/MMPOS/RaveOS is working perfectly
> 
> Ok, thx, will fixed on next release ?

Yes it will, Dev commented it, and he will added in the 1.51 really soon
